src/components/Tooltip/BaseTooltip.js

```diff
@@ -91,10 +132,15 @@ function Tooltip(props) {
         if (bounds.width === 0) {
             setIsRendered(false);
         }
-        setWrapperWidth(bounds.width);
-        setWrapperHeight(bounds.height);
-        setXOffset(bounds.x);
-        setYOffset(bounds.y);
+        // Choose a bounding box for the tooltip to target.
+        // In the case when the target is a link that has wrapped onto
+        // multiple lines, we want to show the tooltip over the part
+        // of the link that the user is hovering over.
+        const betterBounds = chooseBoundingBox(target.current, initialMousePosition.current.x, initialMousePosition.current.y);
+        setWrapperWidth(betterBounds.width);
```

Comment:
I believe this is leading to a crash with `Uncaught TypeError: Cannot read properties of undefined (reading 'width')`. This is what I did:

1. Sign in with a new account
2. Go to Settings > Workspaces > Create New Workspace (don't create anything)
3. Click the back button on the modal
![image](https://github.com/Expensify/App/assets/1228807/7b461ad2-f4c1-447a-9a84-e9218e2b8924)


![image](https://github.com/Expensify/App/assets/1228807/77553a4a-8980-4c78-bb51-881f24d0fbc8)

Reply:
Yes it's, more info https://github.com/Expensify/App/issues/29446#issuecomment-1759942603 and PR fixing it https://github.com/Expensify/App/pull/29468